app/src/main/java/org/fossasia/openevent/api/processor/EventListResponseProcessor.java

```diff
@@ -13,10 +16,36 @@
  */
 public class EventListResponseProcessor implements Callback<EventResponseList> {
     private static final String TAG = "Events";
+
     @Override
     public void success(EventResponseList eventResponseList, Response response) {
+        ArrayList<String> queries = new ArrayList<String>();
```

Comment:
Please note that this call is on the UI Thread. Maybe need to separate it into a separate thread? Or create a base class for the processor and enqueue all the success calls on a separate thread.